io_context/include/io_context/io_context.hpp

```diff
@@ -28,6 +28,8 @@ namespace drivers
 namespace common
 {
 
+//using asio = boost::asio;
```

Comment:
If migrating to Boost.Asio from Asio, it'd be best to remove all these comments

Reply:
Addressed [here](https://github.com/ros-drivers/transport_drivers/pull/84/commits/17df5f06ba3cf8f822d9eecc47e820c1dd5dd188)